src/js/modules/infragistics.ui.tree.js

```diff
@@ -747,7 +747,7 @@
 				//Set
 				settings.allowDrop = {
 					allowDrop: false,
-					```
+					...
```

Comment:
Code snippet should be copy-paste-able. Please don't leave code snippets with `...`